onbuild/Dockerfile

```diff
@@ -0,0 +1,29 @@
+#
+# Dockerfile for Storm Mesos framework
+#
+FROM mesosphere/mesos:0.27.0-0.2.190.ubuntu1404
```

Comment:
Whoa, that's a fancy / nice idea!  I think that would work pretty well.  Thanks for the idea & quick hacking!  I'd be happy to accept a PR for this or do it myself later (a bit busy at moment, per usual).

(BTW, the new Mesosphere site layout looks really nice, *but* it would be great if we could link to the actual thing we've clicked into on [that page](https://mesosphere.com/downloads/#) -- i.e., "Get Started" under Mesos.)